packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js

```diff
@@ -32,7 +32,15 @@ export default class TelecomTelephonyAliasSpecialRepaymentsCtrl {
       })
       .$promise.then((repayments) => {
         this.allRepayments = repayments;
-        this.repayments = repayments.slice(0, LIMIT);
+        this.repayments = repayments
+          .slice(0, LIMIT)
+          .sort(({ creationDatetime: dateA }, { creationDatetime: dateB }) => {
```

Comment:
is it working? don't you have to use moment(date).isAfter()?

Reply:
yes, the dates have the following format 'YYYY-MM-DDTHH:MM:SS'

Reply:
In fact no you're right, some comparison won't work that way for all cases. It could have been possible with 'YYYY-MM_DD' format

Reply:
Sorry for confusion, in the manager, the format is like 'DD-MM-YYYY', but API returns 'YYYY-MM-DD'. So that's why it works.
But I can still secure the test

Reply:
I'm wondering if the sorting couldn't be instead apply directly into the 2API endpoint.
Is that make sense for you?

Reply:
If we consider that these dates should always be sorted this way, yes it could be directly done in the 2api endpoint